src/modules/features/SettingDialogue/settingConstants.ts

```diff
@@ -105,48 +105,50 @@ export const SettingInformation: Partial<Record<keyof AllSettings, Configuration
     },
     syncOnSave: {
         name: "Sync on Save",
-        desc: "When you save a file, sync automatically",
+        desc: "Forces a sync of the file when it's saved.",
     },
     syncOnEditorSave: {
         name: "Sync on Editor Save",
         desc: "When you save a file in the editor, sync automatically",
     },
     syncOnFileOpen: {
         name: "Sync on File Open",
-        desc: "When you open a file, sync automatically",
+        desc: "Forces the file to be synced when opened.",
     },
     syncOnStart: {
-        name: "Sync on Start",
-        desc: "Start synchronization after launching Obsidian.",
+        name: "Sync on Startup",
+        desc: "Automatically Sync all files when opening Obsidian.",
     },
     syncAfterMerge: {
         name: "Sync after merging file",
         desc: "Sync automatically after merging files",
     },
     trashInsteadDelete: {
         name: "Use the trash bin",
-        desc: "Do not delete files that are deleted in remote, just move to trash.",
+        desc: "Move remotely deleted files to the trash, instead of deleting.",
     },
     doNotDeleteFolder: {
         name: "Keep empty folder",
-        desc: "Normally, a folder is deleted when it becomes empty after a synchronization. Enabling this will prevent it from getting deleted",
+        desc: "Should we keep folders that don't have any files inside?",
     },
     resolveConflictsByNewerFile: {
-        name: "Always overwrite with a newer file (beta)",
-        desc: "(Def off) Resolve conflicts by newer files automatically.",
+        name: "(BETA) Always overwrite with a newer file",
+        desc: "Testing only - Resolve file conflicts by syncing newer copies of the file, this can overwrite modified files. Be Warned.",
     },
     checkConflictOnlyOnOpen: {
-        name: "Postpone resolution of inactive files",
+        name: "Delay conflict resolution of inactive files",
+        desc: "Should we only check for conflicts when a file is opened?"
     },
     showMergeDialogOnlyOnActive: {
-        name: "Postpone manual resolution of inactive files",
+        name: "Delay merge conflict prompt for inactive files.",
+        desc: "Should we prompt you about conflicting files when a file is opened?"
     },
     disableMarkdownAutoMerge: {
-        name: "Always resolve conflicts manually",
-        desc: "If this switch is turned on, a merge dialog will be displayed, even if the sensible-merge is possible automatically. (Turn on to previous behavior)",
+        name: "Always prompt merge conflicts",
+        desc: "Should we prompt you for every single merge, even if we can safely merge automatcially?",
     },
     writeDocumentsIfConflicted: {
-        name: "Always reflect synchronized changes even if the note has a conflict",
+        name: "Sync changes even if the file has a conflict.",
```

Comment:
If the file has a conflict, the wording `sync` is not correct. The original `Synchronised` was totally ambiguous. How about `Apply the latest transferred change even if the file has a conflict.` or `Write the latest transferred change even...`?

Reply:
Rewrote these again to the following. 
```
name: "Apply Latest Change if Conflicting",
desc: "Enable this option to automatically apply the most recent change to documents even when it conflicts",
```